src/engine/enginebuffer.cpp

```diff
@@ -1216,6 +1222,13 @@ void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
     // Update indicators that are only updated after every
     // sampleRate/kiUpdateRate samples processed.  (e.g. playposSlider)
     if (m_iSamplesCalculated > (m_pSampleRate->get() / kiPlaypositionUpdateRate)) {
+        m_timeElapsed->set(m_filepos_play / m_iSampleRate / kSamplesPerFrame
```

Comment:
The denominator (m_iSampleRate * kSamplesPerFrame * m_tempo_ratio_old) is needed twice and by precalculating it the code becomes more readable. One division per expression is suffcient.